ArduCopter/version.h

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
-#include "ap_version.h"
+#include "AP_Common/AP_FWVersion.h"
 
 #define THISFIRMWARE "APM:Copter V3.6-dev"
+
+// the following line is parsed by the autotest scripts
 #define FIRMWARE_VERSION 3,6,0,FIRMWARE_VERSION_TYPE_DEV
```

Comment:
As mentioned in the comment - that line is *parsed* by the autotest scripts!

The autotest scripts will not to macro expansion :-)

I *do* want to fix that... somehow... possibly compiling a "version" program.

Something I'd like to do sooner is eliminating `THISFIRMWARE` - that one is entirely doable, assuming the vehicle maintainers are OK with consistency :-)  But that's not for this PR.